### ISSUE TITLE
Wrong translation

### ISSUE THREAD

**Issue by derrapf** [iobroker-community-adapters/ioBroker.snmp#108]
Hi guys
Just wanted to update the adapter and could read the following sentences  in iobroker as well as here on GIT:

```

(McM1957) reduce latency for update of info.connection
(McM1957) avoid excessive error logs if target is unreachable, optimize logging
(McM1957) add additional online at ip base to indicate target is reachable
(McM1957) output warning if OIDs specify different communities for one device
(Apollon77) Add Sentry for crash reporting
```

That is very mistakable!
  "reduce latency for update of info.connection" is an imperative and means that the user should reduce the latency. 
  I guess that is not what you meant.
  It should be "reduced latency.." which means that you! - the developer - reduced the latency.
  The german translation will get worse so.
   It  translated as "Reduzieren Sie die..." which really means that the user should reduce the latency.

and so forth.

So it should be:
reduced latency ..
avoided additional ...
added additional...
output of warning if ...
added Sentry ...

I was really very confused because I thought I have to do all these things....

BR Ralf

**Comment by FredF63** [comment]
Gerade ein PR dafür gestellt

**Comment by mcm1957** [comment]
Thanks - i'll try to improve my english.

**Comment by mcm1957** [comment]
fixed at release 2.0.0
